src/jobs/jobs.service.ts

```
@@ -1,10 +1,14 @@
-
-import { Injectable, NotFoundException, ForbiddenException, Inject } from '@nestjs/common';
+import {
+  Injectable,
+  NotFoundException,
+  ForbiddenException,
+  Inject,
+} from '@nestjs/common';
 
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository, Not, IsNull, DataSource } from 'typeorm';
```

ESLint

[error] 'Not' is defined but never used.

(@typescript-eslint/no-unused-vars)